### DISCUSSION TITLE
Machinaris 8.2 issues with Cactus

### DISCUSSION THREAD

**Discussion by LeroyINC** [guydavis/machinaris#738] [General]
After some playing around i was able to get the Cactus fork work with 8.2

but I noticed some issues on the harvester nodes.

when the docker container starts both the cactus harvester service and Machinaris does not start automatically.
I had to log into the Cactus docker and start them manually.
Also when deploying a fresh worker it did not look like the CA keys were properly copied to the harvester node.

just an FYI

**Comment by guydavis** [comment]
Thanks for the report.  This is fixed, so please check for a `machinaris-cactus` image update, including released stream.  Note, I did have to clear my ~/.machinaris-cactus folder to get things running again though.  Sorry for the trouble this blockchain's SSL certificate expiry has caused.

**Comment by LeroyINC** [comment]
that worked perfectly on that new pull for cactus... this can be closed
